CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
Can you remove this? We no longer do this anymore. You can instead update the original change log to have more details.

Reply:
Updated changelog. This is changelog is now consistent with release changelog in other SDKs.